src/Services/ConvertToQtiService.php

```diff
@@ -169,18 +159,20 @@ private function convertAssessmentItem($json)
     {
         $result = [];
 
-        if (in_array($json['type'], LearnosityExportConstant::$supportedQuestionTypes)) {
-            $result = Converter::convertLearnosityToQtiItem($json);
-        } else {
-            $result = [
-                '',
-                [
-                    'Ignoring ' . $json['type'] . ' , currently unsupported'
-                ]
-            ];
-            $this->output->writeln("  <error>Question type `{$json['type']}` not yet supported, ignoring</error>");
-        }
+        foreach ($json['questions'] as $question):
```

Comment:
Style: This works, but can we please use consistent syntax throughout the codebase? Colon control structures are not used anywhere else, so it is highly preferable we stick with brace syntax here. It also doesn't make a lot of sense that we're mixing it with brace syntax in the same block.

Reply:
Done